__label__NotIgnored failed to get target cluster namespace get https <*> <*> dial tcp <*> <*> connect no route to host
__label__NotIgnored <*> info controller.vmoperator error vm_name variable not set reconciler group <*> reconciler kind vmoperator name <*> namespace <*>
__label__NotIgnored error plugin/errors <*> <*> a read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> hinfo read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> aaaa read udp <*> <*> <*> i/o timeout
__label__NotIgnored failed to get app cluster deployment status error failed to list spc in tenant cluster get https <*> <*> <*> dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to get app cluster resource status error failed to list palette packs get https <*> <*> <*> dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to list spc in tenant cluster get https <*> <*> <*> dial tcp <*> <*> connect no route to host
__label__NotIgnored error failed to load image command docker exec <*> -i <*> ctr <*> <*> images import <*> failed with error exit status <*>
__label__NotIgnored error loading <*> <*> image into kind
__label__NotIgnored  <*> 0m  0m <*> <*>  0m 0m 1mfailed to dial libvirt dial unix <*> connect resource temporarily unavailable 0m
__label__NotIgnored oras push failed for <*> <*> <*> <*> due to exit status <*> <*> error stat <*> no such file or directory n
__label__NotIgnored oras push failed for <*> <*> <*> <*> due to exit status <*> <*> uploading <*> <*> nerror failed commit on ref <*> <*> failed to do request put https <*> <*> digest <*> <*> http/1.x transport connection broken http contentlength <*> with body length <*> n
__label__NotIgnored error plugin/errors <*> <*> hinfo read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> a read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> aaaa read udp <*> <*> <*> i/o timeout
__label__NotIgnored grpc addrconn.createtransport failed to connect to https <*> <*> <nil> 0 <nil> . err connection error desc transport error while dialing dial tcp <*> <*> connect connection refused . reconnecting...
__label__NotIgnored reconciler error error <*> <*> not found name <*> namespace <*> reconciler group <*> reconciler kind machine
__label__NotIgnored <*> <*> <*> error controller.libvirtmachine reconciler error reconciler group <*> reconciler kind libvirtmachine name <*> namespace <*> error <*> <*> not found errorcauses error <*> <*> not found
__label__NotIgnored retrying <*> request get <*> failed request get <*> error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to get create pending cluster from hubble an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to apply pending cluster for creation code unknownerror msg an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host ref details <nil>
__label__NotIgnored failed request get <*> error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed request get <*> servicename <*> error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored retrying <*> request get <*> servicename <*> failed request get <*> servicename <*> error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to fetch latest version for service jet . an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to get latest version for jet an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored upgrade agent failed to check update required in jet an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to check version upgrade an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed request get <*> token error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored retrying <*> request get <*> token failed request get <*> token error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to refresh auth token. an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to refresh auth token. getting new auth token. an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed request post <*> error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored retrying <*> request post <*> failed request post <*> error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to get auth token an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to fetch latest version for service lodge . code requesterror details <nil> message get request failure for with msg get with response caused by get https <*> read tcp <*> <*> <*> read connection reset by peer ref <*>
__label__NotIgnored upgrade agent failed to check update required in lodge code requesterror details <nil> message get request failure for with msg get with response caused by get https <*> read tcp <*> <*> <*> read connection reset by peer ref <*>
__label__NotIgnored failed to check version upgrade code requesterror details <nil> message get request failure for with msg get with response caused by get https <*> read tcp <*> <*> <*> read connection reset by peer ref <*>
__label__NotIgnored reconciler error error failed to reconcile charts failed to reconcile charts string <*> error path <*> not found n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored error plugin/errors <*> get. aaaa read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> get. a read udp <*> <*> <*> i/o timeout
__label__NotIgnored cacher *unstructured.unstructured unexpected listandwatch error failed to list <*> kind issuer conversion webhook for <*> kind issuer failed post https <*> <*> <*> dial tcp <*> <*> connect connection refused reinitializing...
__label__NotIgnored cacher *unstructured.unstructured unexpected listandwatch error failed to list <*> kind certificate conversion webhook for <*> kind certificate failed post https <*> <*> <*> dial tcp <*> <*> connect connection refused reinitializing...
__label__NotIgnored cacher *unstructured.unstructured unexpected listandwatch error failed to list <*> kind certificaterequest conversion webhook for <*> kind certificaterequest failed post https <*> <*> <*> dial tcp <*> <*> connect connection refused reinitializing...
__label__NotIgnored hit error connecting to datastore <*> retry error get https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request post <*> http/1.1 upstream http <*> <*> host <*>
__label__NotIgnored failed to list projects response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for library post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for ceph post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for cephcsi post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored multi-attach error for volume <*> uniquename <*> from node <*> volume is already exclusively attached to node <*> and can t be attached to another
__label__NotIgnored event occurred object <*> kind pod apiversion <*> type warning reason failedattachvolume message multi-attach error for volume <*> volume is already exclusively attached to one node and can t be attached to another
__label__NotIgnored error syncing job internal error occurred failed calling webhook <*> post https <*> <*> <*> dial tcp <*> <*> connect connection refused
__label__NotIgnored event occurred object <*> kind job apiversion batch/v1 type warning reason failedcreate message error creating internal error occurred failed calling webhook <*> post https <*> <*> <*> dial tcp <*> <*> connect connection refused
__label__NotIgnored target has failed health check marking for remediation cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck message failurereason <*> reason machinehasfailure target <*>
__label__NotIgnored failed to reconcile machinehealthcheck error failed to patch unhealthy machine status for machine <*> <*> <*> not found cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck
__label__NotIgnored reconciler error error failed to patch unhealthy machine status for machine <*> <*> <*> not found name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck
__label__NotIgnored failed to reconcile machinehealthcheck error failed to patch unhealthy machine status for machine <*> <*> <*> not found <*> <*> not found cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck
__label__NotIgnored reconciler error error failed to patch unhealthy machine status for machine <*> <*> <*> not found <*> <*> not found name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck
__label__NotIgnored target has failed health check marking for remediation cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck message node failed to report startup in duration duration <*> reason nodestartuptimeout target <*>
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error etcdserver request timed out took too long <*> to execute
__label__NotIgnored operation for volumename <*> podname nodename failed. no retries permitted until <*> <*> <*> <*> <*> utc m <*> durationbeforeretry <*> . error detachvolume.detach failed for volume <*> uniquename <*> on node <*> detach timeout for volume <*>
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored client ip <*> requestid <*> user agent <*> failed to authenticate user _json_key error failed to authenticate user due to error invalid credentials
__label__NotIgnored reconciler error error failed to download pack https <*> <*> error unexpected status code manifests <*> <*> bad gateway n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored reconciler error error failed to download pack https <*> <*> failed to download helm chart for pack ehl-logging version <*> error <*> <*> not found n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored level error time <*> <*> <*> caller vsphere/datacenter.go <*> msg couldn t find vm given uuid traceid <*> stacktrace <*> *datacenter .getvirtualmachinebyuuid n <*> <*> <*> n <*> <*>
__label__NotIgnored error in downloading kind get https <*> dial tcp lookup <*> on <*> <*> no such host
__label__NotIgnored level error time <*> <*> <*> caller node/manager.go <*> msg failed to discover vm with uuid for node <*> traceid <*> stacktrace <*> *defaultmanager .registernode n <*> <*> <*> *nodes .nodeadd n <*> <*> <*> n <*> <*> <*> *processorlistener <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> *processorlistener .run n <*> <*> <*> *group <*> n <*> <*>
__label__NotIgnored failed to reconcile failed to acquire auth token for service edge. an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored failed to get latest edge service version for fetching <*> file failed to acquire auth token for service lodge. an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect no route to host
__label__NotIgnored error running command <*> copy <*> <*> <*> <*> <*> from crane-remote-adapter due to <*> <*> <*> <*> copying from <*> <*> to <*> <*> nerror fetching <*> <*> get https <*> dial tcp lookup <*> on <*> <*> server misbehaving get http <*> dial tcp lookup <*> on <*> <*> server misbehaving n
__label__NotIgnored crane push failed for <*> <*> <*> <*> <*> due to exit status <*>
__label__NotIgnored <*> error controller.vmoperator reconciler error reconciler group <*> reconciler kind vmoperator name <*> namespace <*> error <*> <*> not found
__label__NotIgnored reconciler error error failed to download pack https <*> <*> failed to download helm chart for pack sprsnapshot-service version <*> error <*> <*> not found n exit status <*> name nightly-optimus namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored reconciler error error failed to download pack https <*> <*> failed to download helm chart for pack <*> version <*> error <*> <*> not found n exit status <*> name nightly-optimus namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored failed request get <*> uid error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect connection timed out
__label__NotIgnored failed request get <*> error read connection reset by peer
__label__NotIgnored error canceling upload after error already closed <*> harbor_registry_user go.version <*> http.request.contenttype <*> http.request.host <*> http.request.id <*> http.request.method put http.request.remoteaddr <*> http.request.uri <*> <*> digest <*> http.request.useragent <*> vars.name <*> <*> <*>
__label__NotIgnored response completed with error <*> harbor_registry_user <*> digest invalid err.detail invalid digest for referenced layer sha256 <*> content does not match digest err.message provided digest did not match uploaded content go.version <*> http.request.contenttype <*> http.request.host <*> http.request.id <*> http.request.method put http.request.remoteaddr <*> http.request.uri <*> <*> digest <*> http.request.useragent <*> http.response.contenttype <*> charset utf-8 http.response.duration <*> http.response.status <*> http.response.written <*> vars.name <*> <*> <*>
__label__NotIgnored error running command <*> copy <*> <*> <*> <*> <*> from crane-remote-adapter due to <*> <*> <*> <*> copying from <*> <*> to <*> <*> nerror fetching <*> <*> get https <*> dial tcp lookup <*> on <*> <*> server misbehaving get http <*> dial tcp <*> <*> connect connection refused n
__label__NotIgnored crane push failed for <*> <*> <*> <*> <*> due to exit status <*>
__label__NotIgnored failed to sync chart <*> for https <*> error <*> no such file n exit status <*>
__label__NotIgnored oras push failed for <*> <*> <*> <*> due to exit status <*> <*> error stat <*> no such file or directory n
__label__NotIgnored oras push failed for <*> <*> <*> <*> due to exit status <*> <*> uploading <*> <*> nerror failed commit on ref <*> <*> unexpected status <*> bad request n
__label__NotIgnored error plugin/errors <*> <*> hinfo read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> aaaa read udp <*> <*> <*> i/o timeout
__label__NotIgnored reconciler error error failed to download pack https <*> <*> error unable to fetch descriptor sha256 <*> which reports content size of zero invalid argument n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored failed to get pack object error <*> <*> not found spectrocluster <*>
__label__NotIgnored error running command <*> copy <*> <*> <*> <*> <*> from crane-remote-adapter due to <*> <*> <*> <*> copying from <*> <*> to <*> <*> nerror fetching <*> <*> get https <*> dial tcp lookup <*> on <*> <*> server misbehaving get http <*> dial tcp lookup <*> on <*> <*> server misbehaving n
__label__NotIgnored response completed with error <*> harbor_registry_user <*> blob unknown err.detail sha256 <*> err.message blob unknown to registry go.version <*> http.request.host <*> http.request.id <*> http.request.method head http.request.remoteaddr <*> http.request.uri <*> <*> http.request.useragent <*> http.response.contenttype <*> charset utf-8 http.response.duration <*> http.response.status <*> http.response.written <*> <*> sha256 <*> vars.name <*>
__label__NotIgnored failed to update kubeadmcontrolplane status error failed to create remote cluster client error creating client and cache for remote cluster error creating dynamic rest mapper for remote cluster <*> get https <*> <*> <*> dial tcp <*> <*> connect no route to host cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored reconciler error error failed to create remote cluster client error creating client and cache for remote cluster error creating dynamic rest mapper for remote cluster <*> get https <*> <*> <*> dial tcp <*> <*> connect no route to host name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored failed to update kubeadmcontrolplane status error failed to create remote cluster client error creating client and cache for remote cluster error creating dynamic rest mapper for remote cluster <*> get https <*> <*> <*> <*> request canceled while waiting for connection client.timeout exceeded while awaiting headers cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored reconciler error error failed to create remote cluster client error creating client and cache for remote cluster error creating dynamic rest mapper for remote cluster <*> get https <*> <*> <*> <*> request canceled while waiting for connection client.timeout exceeded while awaiting headers name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request get <*> http/1.1 upstream http <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> <*> http/1.1 upstream http <*> <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request post <*> http/1.1 upstream http <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request get <*> <*> page_size <*> with_detail true http/1.1 upstream http <*> <*> <*> page_size <*> with_detail true host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> sig-storage http/1.1 upstream http <*> <*> sig-storage host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> coredns http/1.1 upstream http <*> <*> coredns host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> kubebuilder http/1.1 upstream http <*> <*> kubebuilder host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> k8scsi http/1.1 upstream http <*> <*> k8scsi host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> http/1.1 upstream http <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> cephcsi http/1.1 upstream http <*> <*> cephcsi host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> rook http/1.1 upstream http <*> <*> rook host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> ceph http/1.1 upstream http <*> <*> ceph host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> charts http/1.1 upstream http <*> <*> charts host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request get <*> http/1.1 upstream http <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> bitnami http/1.1 upstream http <*> <*> bitnami host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request head <*> http/1.1 upstream http <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> kindest http/1.1 upstream http <*> <*> kindest host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> google_containers http/1.1 upstream http <*> <*> google_containers host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request get <*> <*> page_size <*> with_detail true http/1.1 upstream http <*> <*> <*> page_size <*> with_detail true host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> connect failed <*> connection refused while connecting to upstream client <*> server request head <*> rancher http/1.1 upstream http <*> <*> rancher host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request head <*> <*> http/1.1 upstream http <*> <*> <*> host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request head <*> ceph http/1.1 upstream http <*> <*> ceph host <*>
__label__NotIgnored <*> <*> <*> <*> error <*> <*> upstream timed out <*> connection timed out while connecting to upstream client <*> server request head <*> sig-storage http/1.1 upstream http <*> <*> sig-storage host <*>
__label__NotIgnored create project failed for <*> post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored error building harbor content map response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for k8scsi post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for sig-storage post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for kindest post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for rancher post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for kubebuilder post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for bitnami post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for google_containers post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for charts post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for coredns post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored create project failed for rook post <*> <*> createprojectcreated response status code does not match any response statuses defined for this endpoint in the swagger spec status <*>
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error context deadline exceeded took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error context canceled took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> with result error context canceled took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> count_only true with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> limit <*> with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> count_only true with result error etcdserver request timed out took too long <*> to execute
__label__NotIgnored aug <*> <*> <*> <*> error overlord_read_repo.go <*> *mongooverlordreadrepo .findoverlordbyname reqid <*> finding the overlord by name failed. resourcenotfound private gateway matching value <*> system private gateway is not found
__label__NotIgnored aug <*> <*> <*> <*> error metadata_service.go <*> *metadataservice .updatelastmodifiedtime failed to update last modified timestamp for installer <*> due to dbupdateerror update to installers failed caused by mongo no documents in result
__label__NotIgnored multi-attach error for volume <*> uniquename <*> from node <*> volume is already exclusively attached to node <*> and can t be attached to another
__label__NotIgnored event occurred object <*> kind pod apiversion <*> type warning reason failedattachvolume message multi-attach error for volume <*> volume is already exclusively attached to one node and can t be attached to another
__label__NotIgnored multi-attach error for volume <*> uniquename <*> from node <*> volume is already used by pods <*> on node <*>
__label__NotIgnored event occurred object <*> kind pod apiversion <*> type warning reason failedattachvolume message multi-attach error for volume <*> volume is already used by pod s <*>
__label__NotIgnored error removing node <*> from <*> failed to delete node <*> from list of nodes managed by attach/detach <*> node still contains <*> volumes in its list of volumes to attach
__label__NotIgnored <*> detachment for volumeattachment for volume <*> failed rpc error code internal desc failed to find virtualmachine for node <*> . error node wasn t found
__label__NotIgnored operation for volumename <*> podname nodename failed. no retries permitted until <*> <*> <*> <*> <*> utc m <*> durationbeforeretry <*> . error detachvolume.detach failed for volume <*> uniquename <*> on node <*> rpc error code internal desc failed to find virtualmachine for node <*> . error node wasn t found
__label__NotIgnored <*> detachment for volumeattachment for volume <*> failed rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored <*> detachment for volumeattachment for volume <*> failed rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err serverfaultcode notauthenticated
__label__NotIgnored <*> attachment for <*> failed rpc error code internal desc failed to find virtualmachine for node <*> . error post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored operation for volumename <*> podname nodename failed. no retries permitted until <*> <*> <*> <*> <*> utc m <*> durationbeforeretry <*> . error attachvolume.attach failed for volume <*> uniquename <*> from node <*> rpc error code internal desc failed to find virtualmachine for node <*> . error post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored event occurred object <*> kind pod apiversion <*> type warning reason failedattachvolume message attachvolume.attach failed for volume <*> rpc error code internal desc failed to find virtualmachine for node <*> . error post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored operation for volumename <*> podname nodename failed. no retries permitted until <*> <*> <*> <*> <*> utc m <*> durationbeforeretry <*> . error attachvolume.attach failed for volume <*> uniquename <*> from node <*> attach timeout for volume <*>
__label__NotIgnored event occurred object <*> kind pod apiversion <*> type warning reason failedattachvolume message attachvolume.attach failed for volume <*> attach timeout for volume <*>
__label__NotIgnored <*> detachment for volumeattachment for volume <*> failed rpc error code deadlineexceeded desc context deadline exceeded
__label__NotIgnored operation for volumename <*> podname nodename failed. no retries permitted until <*> <*> <*> <*> <*> utc m <*> durationbeforeretry <*> . error detachvolume.detach failed for volume <*> uniquename <*> on node <*> rpc error code deadlineexceeded desc context deadline exceeded
__label__NotIgnored <*> attacher.volumesareattached failed for attach.id <*> <*> <*> not found
__label__NotIgnored <*> detachment for volumeattachment for volume <*> failed rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> context deadline exceeded
__label__NotIgnored operation for volumename <*> podname nodename failed. no retries permitted until <*> <*> <*> <*> <*> utc m <*> durationbeforeretry <*> . error detachvolume.detach failed for volume <*> uniquename <*> on node <*> rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> context deadline exceeded
__label__NotIgnored <*> detachment for volumeattachment for volume <*> failed rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout
__label__NotIgnored error while looking for vm <*> byname in vc <*> and datacenter <*> no vm found
__label__NotIgnored error while looking for vm <*> byip in vc <*> and datacenter <*> no vm found
__label__NotIgnored failed to <*> detach error to <*> <*> <*> not found
__label__NotIgnored error processing <*> failed to detach rpc error code internal desc failed to find virtualmachine for node <*> . error node wasn t found
__label__NotIgnored error processing <*> failed to detach rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored error processing <*> failed to attach rpc error code internal desc failed to find virtualmachine for node <*> . error post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored error processing <*> failed to attach rpc error code deadlineexceeded desc context deadline exceeded
__label__NotIgnored error processing <*> failed to detach rpc error code deadlineexceeded desc context deadline exceeded
__label__NotIgnored error processing <*> failed to detach rpc error code internal desc queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids dynamicdata id <*> names containerclusterids storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> context deadline exceeded
__label__NotIgnored failed to get latest edge service version for fetching <*> file code requesterror details <nil> message get request failure for with msg get with response caused by get https <*> read tcp <*> <*> <*> read connection reset by peer ref <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg failed to obtain user session with err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout traceid <*> stacktrace <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .getdatacenters n <*> <*> <*> n <*> <*> <*> *authmanager <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg cannot connect to vcenter with err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout traceid <*> stacktrace <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .getdatacenters n <*> <*> <*> n <*> <*> <*> *authmanager <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg could not logout of vc session. error post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout traceid <*> stacktrace <*> *virtualcenter <*> n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .getdatacenters n <*> <*> <*> n <*> <*> <*> *authmanager <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg failed to connect to vcenter. err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout traceid <*> stacktrace <*> *virtualcenter .getdatacenters n <*> <*> <*> n <*> <*> <*> *authmanager <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller common/authmanager.go <*> msg failed to get datacenter list. err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout traceid <*> stacktrace <*> n <*> <*> <*> *authmanager <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg failed to obtain user session with err post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg cannot connect to vcenter with err post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg could not logout of vc session. error post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> *virtualcenter <*> n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg failed to connect to virtual center host <*> with err post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller volume/manager.go <*> msg connectcns failed with err post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg pvcupdated queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp <*> <*> i/o timeout stacktrace <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level info time <*> <*> <*> caller syncer/metadatasyncer.go <*> msg csi full sync failed with error rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp <*> <*> i/o timeout
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg could not logout of vc session. error serverfaultcode the session is not authenticated. stacktrace <*> *virtualcenter <*> n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg failed to obtain user session with err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg cannot connect to vcenter with err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualcenter.go <*> msg could not logout of vc session. error post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> *virtualcenter <*> n <*> <*> <*> *virtualcenter .connect n <*> <*> <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg failed to connect to virtual center host <*> with err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> *virtualcenter .connectcns n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller volume/manager.go <*> msg connectcns failed with err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg pvcupdated queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout stacktrace <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level info time <*> <*> <*> caller syncer/metadatasyncer.go <*> msg csi full sync failed with error rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err post https <*> <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout
__label__NotIgnored delete <*> volume deletion failed persistentvolume <*> is still attached to node <*>
__label__NotIgnored error syncing volume <*> persistentvolume <*> is still attached to node <*>
__label__NotIgnored event <*> kind persistentvolume namespace name <*> uid <*> apiversion <*> resourceversion <*> fieldpath type warning reason volumefaileddelete persistentvolume <*> is still attached to node <*>
__label__NotIgnored level error time <*> <*> <*> caller vsphere/datacenter.go <*> msg couldn t find vm given uuid <*> traceid <*> stacktrace <*> *datacenter .getvirtualmachinebyuuid n <*> <*> <*> n <*> <*>
__label__NotIgnored mount failed args <*> ext4 <*> defaults <*> <*> cmd mount error exit status <*> output mount <*> wrong fs type bad option bad superblock on <*> missing codepage or helper program or other error. n
__label__NotIgnored target has failed health check marking for remediation cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck message reason nodenotfound target <*>
__label__NotIgnored reconciler error error post https <*> dial tcp lookup <*> on <*> <*> server misbehaving name <*> namespace <*> reconciler group <*> reconciler kind vspherevm
__label__NotIgnored reconciler error error unexpected error while probing vcenter for <*> kind vspherecluster <*> post https <*> dial tcp lookup <*> on <*> <*> server misbehaving name <*> namespace <*> reconciler group <*> reconciler kind vspherecluster
__label__NotIgnored unable to create session error post https <*> dial tcp lookup <*> on <*> <*> server misbehaving vspheredeploymentzone <*>
__label__NotIgnored reconciler error error unable to create auth session post https <*> dial tcp lookup <*> on <*> <*> server misbehaving name <*> namespace reconciler group <*> reconciler kind vspheredeploymentzone
__label__NotIgnored retrying <*> request post <*> failed request post <*> error an unknown error occurred. proxyconnect tcp dial tcp lookup <*> on <*> <*> server misbehaving
__label__NotIgnored failed to acquire auth token for service ally. code unknownerror msg an unknown error occurred. proxyconnect tcp dial tcp lookup <*> on <*> <*> server misbehaving ref details <nil> cluster <*>
__label__NotIgnored failed to get auth token for service ally. an unknown error occurred. proxyconnect tcp dial tcp lookup <*> on <*> <*> server misbehaving cluster <*>
__label__NotIgnored reconciler error error failed to download pack https <*> <*> error unexpected status code manifests <*> <*> gateway time-out n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to check if ova is deployed unable to find virtual machine failed to get govmomi finder with datacenter <*>
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to check if ova is deployed unable to find virtual machine failed to get govmomi finder with datacenter <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored failed to keep alive govmomi client error post https <*> dial tcp lookup <*> on <*> <*> server misbehaving
__label__NotIgnored failed to create govmomi client spectrocluster <*>
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to initialize govc manager post https <*> dial tcp lookup <*> on <*> <*> server misbehaving
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to initialize govc manager post https <*> dial tcp lookup <*> on <*> <*> server misbehaving name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to initialize govc manager post https <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to initialize govc manager post https <*> dial tcp lookup <*> on <*> <*> read udp <*> <*> <*> i/o timeout name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored error running command <*> auth login <*> <*> <*> <*> from crane-remote-adapter due to error username and password required n
__label__NotIgnored crane auth login failed <*> <*> due to exit status <*>
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> with result error etcdserver request timed out took too long <*> to execute
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualmachine.go <*> msg returning vm not found err for uuid traceid <*> stacktrace <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager .registernode n <*> <*> <*> *nodes .nodeadd n <*> <*> <*> n <*> <*> <*> *processorlistener <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> *processorlistener .run n <*> <*> <*> *group <*> n <*> <*>
__label__NotIgnored waiting for control plane to pass preflight checks cluster-name <*> name <*> namespace <*> failures machine <*> does not have apiserverpodhealthy condition machine <*> does not have controllermanagerpodhealthy condition machine <*> does not have schedulerpodhealthy condition machine <*> does not have etcdpodhealthy condition machine <*> does not have etcdmemberhealthy condition machine <*> reports apiserverpodhealthy condition is unknown failed to get pod status machine <*> reports controllermanagerpodhealthy condition is unknown failed to get pod status
__label__NotIgnored retrying <*> request get <*> uid failed request get <*> uid error an unknown error occurred. proxyconnect tcp dial tcp <*> <*> connect connection timed out
__label__NotIgnored error spectrocluster namespace <*> name <*> message warning <*> <*> is deprecated in <*> unavailable in <*> use <*> <*>
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored  <*> 0m  0m <*> <*>  0m 0m <*> defining libvirt domain operation failed domain <*> already exists with uuid <*> 0m
__label__NotIgnored oras push failed for <*> <*> <*> <*> due to exit status <*> <*> uploading <*> <*> nerror failed commit on ref <*> <*> failed to do request put https <*> <*> digest <*> <*> http/1.x transport connection broken http contentlength <*> with body length <*> n
__label__NotIgnored  <*> 0m  0m <*> <*>  0m 0m <*> defining libvirt domain operation failed domain standby-gateway already exists with uuid <*> 0m
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error etcdserver request timed out took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> count_only true with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> count_only true with result error etcdserver request timed out took too long <*> to execute
__label__NotIgnored level error time <*> <*> <*> caller vsphere/virtualmachine.go <*> msg returning vm not found err for uuid traceid <*> stacktrace <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager .registernode n <*> <*> <*> *nodes .nodeadd n <*> <*> <*> n <*> <*> <*> *processorlistener <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> *processorlistener .run n <*> <*> <*> *group <*> n <*> <*>
__label__NotIgnored error retrieving resource lock <*> rpc error code unknown desc ok http status code <*> transport missing content-type field
__label__NotIgnored level error time <*> <*> <*> caller volume/manager.go <*> msg cns queryallvolume failed from vcenter <*> with err serverfaultcode nopermission stacktrace <*> *defaultmanager <*> n <*> <*> <*> *defaultmanager <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err serverfaultcode nopermission stacktrace <*> n <*> <*> <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level error time <*> <*> <*> caller <*> <*> msg pvcupdated queryvolume failed with err rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err serverfaultcode nopermission stacktrace <*> n <*> <*> <*> n <*> <*>
__label__NotIgnored level info time <*> <*> <*> caller syncer/metadatasyncer.go <*> msg csi full sync failed with error rpc error code internal desc queryallvolume failed for queryfilter dynamicdata volumeids names containerclusterids <*> storagepolicyid datastores labels compliancestatus datastoreaccessibilitystatus cursor <nil> healthstatus . err serverfaultcode nopermission
__label__NotIgnored error syncing claim <*> failed to provision volume with storageclass <*> rpc error code internal desc failed to create volume. error serverfaultcode nopermission
__label__NotIgnored event <*> kind persistentvolumeclaim namespace <*> name <*> uid <*> apiversion <*> resourceversion <*> fieldpath type warning reason provisioningfailed failed to provision volume with storageclass <*> rpc error code internal desc failed to create volume. error serverfaultcode nopermission
__label__NotIgnored reconciler error error failed to reconcile charts failed to reconcile charts string <*> unable to install chart <*> error installation failed failed post-install timed out waiting for the condition n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored failed to list payload for get <*> project_name <*> repository_name <*> <*> listartifactsinternalservererror errors <*>
__label__NotIgnored  <*> 0m  0m <*> <*>  0m 0m <*> defining libvirt domain operation failed domain active-gateway already exists with uuid <*> 0m
__label__NotIgnored <*> <*> <*> <*> utc <*> error duplicate key value violates unique constraint project_name_key
__label__NotIgnored failed to unmarshal pack content for pack <*> error unmarshaling json json cannot unmarshal string into go struct field content.pack.content.images of type domain.images cluster <*>
__label__NotIgnored failed to get pack content error unmarshaling json json cannot unmarshal string into go struct field content.pack.content.images of type domain.images cluster <*>
__label__NotIgnored successfully audited failed to get pack content error unmarshaling json json cannot unmarshal string into go struct field content.pack.content.images of type domain.images error cluster <*>
__label__NotIgnored failed to run orchestration task error unmarshaling json json cannot unmarshal string into go struct field content.pack.content.images of type domain.images cluster <*>
__label__NotIgnored executing command failed error exit status <*> cmd virsh <*> <*> out error command <*> requires <domain> option n
__label__NotIgnored response completed with error <*> harbor_registry_user <*> blob unknown err.detail sha256 <*> err.message blob unknown to registry go.version <*> http.request.host <*> http.request.id <*> http.request.method get http.request.remoteaddr <*> http.request.uri <*> <*> <*> http.request.useragent <*> http.response.contenttype <*> charset utf-8 http.response.duration <*> http.response.status <*> http.response.written <*> <*> sha256 <*> vars.name <*>
__label__NotIgnored reconciler error error failed to reconcile charts failed to fetch the list of deleted chart releases unable to get helm release list for pack prometheus-operator error path <*> not found n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored reconciler error error failed to reconcile charts failed to fetch the list of deleted chart releases failed to get all the files in dir <*> stat <*> no such file or directory name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored failed to update config map request entity too large limit is <*>
__label__NotIgnored error populating config map request entity too large limit is <*>
__label__NotIgnored retrying <*> request get <*> uid <*> manifestuid failed request get <*> uid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request patch <*> uid <*> failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> failed request get <*> uid <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> failed request patch <*> uid <*> error read connection reset by peer
__label__NotIgnored failed request patch <*> uid <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> failed request get <*> uid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> manifestuid failed request get <*> registryname error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> registryname failed request get <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> failed request get <*> uid <*> error read connection reset by peer
__label__NotIgnored retrying <*> request patch <*> uid <*> retrying <*> request get <*> uid <*> failed request get <*> uid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid failed request get <*> uid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid failed request get <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> failed request get <*> token error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> token failed request get <*> error read connection reset by peer
__label__NotIgnored retrying <*> request patch <*> uid <*> retrying <*> request get <*> uid <*> manifestuid failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> failed request get <*> uid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> manifestuid failed request get <*> uid <*> error read connection reset by peer
__label__NotIgnored target has failed health check marking for remediation cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind machinehealthcheck message failurereason <*> reason machinehasfailure target <*>
__label__NotIgnored reconciler error error failed to retrieve libvirtmachinetemplate external object <*> <*> <*> etcdserver leader changed name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored workers have error details machine <*> errorreason createerror errormessage failure detected from referenced resource <*> kind libvirtmachine with name <*> error creating virtual machine error creating a new managed disk <*> <*> true <*> error creating target volume virerror code <*> domain <*> message requested operation is not valid storage pool <*> is not active
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> limit <*> with result error etcdserver request timed out took too long <*> to execute
__label__NotIgnored error plugin/errors <*> <*> ptr read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> srv read udp <*> <*> <*> i/o timeout
__label__NotIgnored failed to initialize control plane error failed to retrieve infra obj for machine <*> failed to retrieve vspheremachine external object <*> <*> <*> etcdserver request timed out cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored failed to update kubeadmcontrolplane status error failed to retrieve infra obj for machine <*> failed to retrieve vspheremachine external object <*> <*> <*> etcdserver request timed out cluster <*> name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored reconciler error error failed to retrieve vspheremachinetemplate external object <*> <*> <*> etcdserver request timed out failed to retrieve infra obj for machine <*> failed to retrieve vspheremachine external object <*> <*> <*> etcdserver request timed out name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored failed request get <*> error code service unavailable details <html>
__label__NotIgnored retrying <*> request get <*> failed request get <*> error code service unavailable details <html>
__label__NotIgnored failed to get create pending cluster from hubble code service unavailable details <html> r n<head><title>503 service temporarily unavailable</title></head> r n<body> r <*> service temporarily <*> r n<hr><center>nginx</center> r n</body> r <*> r n message <*> service temporarily unavailable ref
__label__NotIgnored failed to apply pending cluster for creation code service unavailable msg <*> service temporarily unavailable ref details <html> r n<head><title>503 service temporarily unavailable</title></head> r n<body> r <*> service temporarily <*> r n<hr><center>nginx</center> r n</body> r <*> r n
__label__NotIgnored failed to apply pending cluster task code service unavailable msg <*> service temporarily unavailable ref details <html> r n<head><title>503 service temporarily unavailable</title></head> r n<body> r <*> service temporarily <*> r n<hr><center>nginx</center> r n</body> r <*> r n
__label__NotIgnored reconciler error error failed to reconcile charts failed to fetch the list of deleted chart releases unable to get helm release list for pack prometheus-operator error path <*> not found n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored reconciler error error failed to reconcile charts failed to fetch the list of deleted chart releases unable to get helm release list for pack <*> error path <*> not found n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored reconciler error error failed to reconcile charts failed to fetch the list of deleted chart releases failed to get all the files in dir <*> stat <*> no such file or directory name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored reconciler error error failed to reconcile charts failed to reconcile charts string <*> unable to update chart <*> error upgrade failed another operation <*> is in progress n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored error plugin/errors <*> <*> a read udp <*> <*> <*> i/o timeout
__label__NotIgnored drain failed retry in <*> error error when evicting pod <*> global timeout reached <*> cluster <*> name <*> namespace <*> node <*> reconciler group <*> reconciler kind machine
__label__NotIgnored error when evicting pod <*> will retry after <*> cannot evict pod as it would violate the pod s disruption budget.
__label__NotIgnored drain failed retry in <*> error error when evicting pod <*> post https <*> <*> <*> context deadline exceeded cluster <*> name <*> namespace <*> node <*> reconciler group <*> reconciler kind machine
__label__NotIgnored failed request get <*> servicename <*> error code service unavailable details <html>
__label__NotIgnored retrying <*> request get <*> servicename <*> failed request get <*> servicename <*> error code service unavailable details <html>
__label__NotIgnored failed request get <*> token error code service unavailable details <html>
__label__NotIgnored retrying <*> request get <*> token failed request get <*> token error code service unavailable details <html>
__label__NotIgnored failed to get latest edge service version for fetching <*> file code service unavailable details <html> r n<head><title>503 service temporarily unavailable</title></head> r n<body> r <*> service temporarily <*> r n<hr><center>nginx</center> r n</body> r <*> r n message <*> service temporarily unavailable ref
__label__NotIgnored retrying <*> request get <*> failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> manifestuid failed request get <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> failed request get <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> failed request get <*> registryname error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> registryname failed request get <*> uid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid failed request get <*> uid <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> failed request get <*> token error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> token failed request patch <*> uid <*> error read connection reset by peer
__label__NotIgnored retrying <*> request patch <*> uid <*> failed request get <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> manifestuid failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> registryname failed request get <*> registryname error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> registryname failed request patch <*> uid <*> error read connection reset by peer
__label__NotIgnored retrying <*> request patch <*> uid <*> failed request get <*> registryname error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> registryname failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored failed to fetch manifest with uid <*> for pack <*> read connection reset by peer
__label__NotIgnored failed request get <*> uid <*> manifestuid error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> uid <*> manifestuid failed request patch <*> uid <*> error read connection reset by peer
__label__NotIgnored reconciler error error failed to download pack https <*> <*> failed to download helm chart for pack ehl-logging version <*> error <*> <*> not found n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored reconciler error error failed to download pack https <*> <*> error unable to fetch descriptor sha256 <*> which reports content size of zero invalid argument n exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored oras push failed for <*> <*> <*> <*> due to exit status <*> <*> error stat <*> no such file or directory n
__label__NotIgnored error plugin/errors <*> <*> hinfo read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> a read udp <*> <*> <*> i/o timeout
__label__NotIgnored error plugin/errors <*> <*> aaaa read udp <*> <*> <*> i/o timeout
__label__NotIgnored failed to get pack object error <*> <*> not found spectrocluster <*>
__label__NotIgnored failed request post <*> error code service unavailable details <html>
__label__NotIgnored retrying <*> request post <*> failed request post <*> error code service unavailable details <html>
__label__NotIgnored failed to get auth token for service jet. code service unavailable details <html> r n<head><title>503 service temporarily unavailable</title></head> r n<body> r <*> service temporarily <*> r n<hr><center>nginx</center> r n</body> r <*> r n message <*> service temporarily unavailable ref
__label__NotIgnored failed to get auth token code service unavailable details <html> r n<head><title>503 service temporarily unavailable</title></head> r n<body> r <*> service temporarily <*> r n<hr><center>nginx</center> r n</body> r <*> r n message <*> service temporarily unavailable ref
__label__NotIgnored failed request get <*> error code service unavailable details <html>
__label__NotIgnored retrying <*> request get <*> failed request get <*> error code service unavailable details <html>
__label__NotIgnored rejected by webhook <*> errors.statuserror errstatus <*> typemeta <*> kind apiversion listmeta <*> selflink resourceversion continue remainingitemcount <*> nil status failure message admission webhook <*> denied the request spec forbidden vspherefailuredomainspec is immutable reason spec forbidden vspherefailuredomainspec is immutable details <*> nil code <*>
__label__NotIgnored aug <*> <*> <*> <*> error email_client.go <*> dial email failed with the error connectproxy <*> status <*> bad request
__label__NotIgnored <*> <*> <*> i control logicalsessioncachereap failed to reap transaction table notyetinitialized replication has not yet been configured
__label__NotIgnored reconciler error error failed to reconcile manifests failed to do kubectl apply unable to connect to the server context deadline exceeded client.timeout exceeded while awaiting headers exit status <*> name <*> namespace <*> reconciler group <*> reconciler kind pack
__label__NotIgnored <*> <*> <*> <*>  <*> 0m failed to connect to nats nats no servers available for connection
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error context deadline exceeded took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error context canceled took too long <*> to execute
__label__NotIgnored <*> <*> <*> <*> w | etcdserver read-only range request key <*> range_end <*> limit <*> with result error etcdserver leader changed took too long <*> to execute
__label__NotIgnored level error time <*> <*> <*> caller node/manager.go <*> msg failed to discover vm with uuid for node <*> traceid <*> stacktrace <*> *defaultmanager .registernode n <*> <*> <*> *nodes .nodeadd n <*> <*> <*> n <*> <*> <*> *processorlistener <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> n <*> <*> <*> *processorlistener .run n <*> <*> <*> *group <*> n <*> <*>
__label__NotIgnored failed running prebind plugin err binding volumes timed out waiting for the condition plugin volumebinding pod <*>
__label__NotIgnored error scheduling pod retrying err running prebind plugin volumebinding binding volumes timed out waiting for the condition pod <*>
__label__NotIgnored error while looking for vm <*> byname in vc <*> and datacenter bellevue dell no vm found
__label__NotIgnored error while looking for vm <*> byip in vc <*> and datacenter bellevue dell no vm found
__label__NotIgnored reconciler error error failed to retrieve vspheremachinetemplate external object <*> <*> <*> etcdserver request timed out name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored retrying <*> request get <*> failed request patch <*> uid <*> error read connection reset by peer
__label__NotIgnored should not happen failed to update managedfields err failed to convert new object <*> kind replicaset to smd typed errors n <*> name <*> <*> duplicate entries for key name ssa_bios_keyname n <*> name <*> <*> duplicate entries for key name ilo_base_path n <*> name <*> <*> duplicate entries for key name ssa_base_path versionkind <*> kind
__label__NotIgnored should not happen failed to update managedfields err failed to convert new object <*> kind deployment to smd typed <*> name <*> <*> duplicate entries for key name releasenamespace versionkind <*> kind
__label__NotIgnored should not happen failed to update managedfields err failed to convert new object <*> kind replicaset to smd typed <*> name <*> <*> duplicate entries for key name store_threadpool_size versionkind <*> kind
__label__NotIgnored should not happen failed to update managedfields err failed to convert new object <*> kind deployment to smd typed errors n <*> name <*> <*> duplicate entries for key name ssa_bios_keyname n <*> name <*> <*> duplicate entries for key name ilo_base_path n <*> name <*> <*> duplicate entries for key name ssa_base_path versionkind <*> kind
__label__NotIgnored should not happen failed to update managedfields err failed to convert new object <*> kind deployment to smd typed <*> name <*> <*> duplicate entries for key name store_threadpool_size versionkind <*> kind
__label__NotIgnored reconciler error error failed to retrieve libvirtmachinetemplate external object <*> <*> <*> etcdserver request timed out name <*> namespace <*> reconciler group <*> reconciler kind kubeadmcontrolplane
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to import image <*> to host <*> failed to get libvirt identity endpoint failed to get ssh secret for host <*> secret not found
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to update cloudconfigstatus failed to import image <*> to host <*> failed to get libvirt identity endpoint failed to get ssh secret for host <*> secret not found name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored condition spectrocluster <*> condition type imagecustomizationdone status false lastprobetime <*> <*> <*> lasttransitiontime <*> <*> <*> reason failure message failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory
__label__NotIgnored reconciler error error failed to create image on cloud failed to create spectro-ansible playbook failed to locate pack stat <*> no such file or directory name <*> namespace <*> reconciler group <*> reconciler kind spectrocluster
__label__NotIgnored failed request get <*> servicename <*> error read connection reset by peer
__label__NotIgnored retrying <*> request get <*> servicename <*> failed request get <*> servicename <*> error read connection reset by peer
__label__NotIgnored <*> <*> <*> <*> <*> <*> <*> w | etcdserver read-only range request key <*> with result error context canceled took too long <*> to execute
__label__NotIgnored reconciler error error error creating azuremanagedcontrolplane <*> failed to reconcile managed cluster resource group failed to create resource <*> service group failed to get desired parameters for group <*> failed to get rg <*> resources.groupsclient#get failure sending request statuscode 0 <*> original error context deadline exceeded name tf-test namespace <*> reconciler group <*> reconciler kind azuremanagedcontrolplane
